cv.py

```diff
@@ -0,0 +1,377 @@
+        #high_white = np.array([123 + 10, 22.95 + 20, 181.05 + 30]);
+        #mask = cv2.inRange(hsv, low_white, high_white);
+
+        #green thresholds
+        #180, 17, 100
+        #180, 1, 100
+        #low_green = np.array([50, 50, 50])
+        #high_green = np.array([90, 63.75, 255])
+        low_green = np.array([60, 90.0, 50.0]) 
+        high_green = np.array([87, 255, 229.0])
+
+        #makes mask
+        mask = cv2.inRange(hsv, low_green, high_green)
+
+        #shows mask
+        cv2.imshow("Mask", mask)
+
+        #find contours based on mask
+        _, contour,_ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
+
+        #initializes rectangle variables with a dummy value
+        max_area = 0
+        mx,my,mw,mh = 0, 0, 0, 0 #variables x,y,width,height associated with the biggest rectangle
+
+        secmax_area = 0
+        sx,sy,sw,sh = 0, 0, 0, 0 #variables x,y,width,height associated with the second biggest rectangle
+
+        thirdmax_area = 0
+        tx,ty,tw,th = 0, 0, 0, 0 #variables x,y,width,height associated with the third biggest rectangle
+
+        #initializes rectangle variables for side cases
+        modmx, modmy, modmw, modmh = mx, my, mw, mh
+        modsx, modsy, modsw, modsh = sx, sy, sw, sh
+
+        #finds the two biggest rectangles
+        for cnt in contour:
+            #gets the x, y of the top left corner and 
+            #width and height of the rectangle
+            x,y,w,h = cv2.boundingRect(cnt)
+            #cv2.rectangle(frame,(x,y),(x+w,y+h),(0,150,255),thickness=3)
+
+            #gets area of the rectangle
+            area = cv2.contourArea(cnt)
+            #print str(area)
+
+            #checks if the rectangle is greater than the max_area so far and
+            #sets the variables to accommodate the change
+            if(area >= max_area):
+                thirdmax_area = secmax_area
+                secmax_area = max_area
+                max_area = area
+                
+                tx = sx
+                ty = sy
+                tw = sw
+                th = sh
+
+                sx = mx
+                sy = my
+                sw = mw
+                sh = mh
+
+                mx = x
+                my = y
+                mw = w
+                mh = h
+            #checks if the rectangle is greater than the secmax_area so far and
+            #sets the variables to accommodate the change
+            elif(area >= secmax_area):
+                thirdmax_area = secmax_area
+                secmax_area = area
+                
+                tx = sx
+                ty = sy
+                tw = sw
+                th = sh
+
+                sx = x
+                sy = y
+                sw = w
+                sh = h
+            #checks if the rectangle is greater than the thirdmax_area so far and
+            #sets the variables to accommodate the change
+            elif(area >= thirdmax_area):
```

Comment:
I think the third rectangle case doesn't apply this year because this case was relevant when the peg was blocking through the middle of a rectangle, but I don't think there's any game piece that gets in the way this year.